README.md

```diff
@@ -1,3 +1,295 @@
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-d
+    credentials:                               # credentials to get access to the seed cluster through service account
+
+  <a href="#landscapeetcd">etcd</a>:
+    backup:
+      type: &lt;gcs|s3&gt;                          # type of blob storage
+      resourceGroup:                          # Azure specific, see below
+      region: (( iaas.region ))               # region of blob storage (default: same as above)
+      credentials: (( iaas.credentials ))     # credentials for the blob storage's IaaS provider (default: same as above)
+
+  <a href="#landscapedns">dns</a>:
+    type: &lt;google-clouddns|aws-route53&gt;       # dns provider
+    hostedZoneID:                             # hosted zone id for chosen domain
+    credentials:                              # credentials for the dns provider
+
+  <a href="#landscapeidentity">identity</a>:
+    users:
+      - email:                                # email (used for Gardener dashboard login)
+        username:                             # username (displayed in Gardener dashboard)
+        password:                             # clear-text password (used for Gardener dashboard login)
+      - email:                                # see above
+        username:                             # see above
+        hash:                                 # bcrypted hash of password, see above
+</pre>
+
+
+### landscape.name
+```yaml
+landscape:
+  name: <Identifier>                       # general Gardener landscape identifier, for example, `my-gardener`
+```
+Arbitrary name for your landscape. The name will be part of the names for resources, for example, the etcd buckets.
+
+### landscape.cluster
+```yaml
+cluster:                                            # Information about your base cluster
+  kubeconfig: <relative path + filename>            # Path to your `kubeconfig` file, relative to folder `landscape`   
+  domain: <prefix>.<cluster domain>                 # Unique basis domain for DNS entries
+  iaas: <gcp|aws|azure>                             # IaaS provider (coming soon: openstack|alicloud)
+  region: <major region>-<minor region>             # Example (gcp, aws): europe-west1; example (Azure): westeurope
+```
+
+Information about your base cluster, where the Gardener will be deployed on.
+
+| Field | Type | Description | Example |
+|:------|:--------|:--------|:--------|
+|`kubeconfig`|File path| Path to your kubeconfig, relative to your landscape folder. It is recommended to create a kubeconfig file in your landscape folder to be able to sync all files relevant for your installation with a git repository.| `./kubeconfig` |
+|`domain`| Unique name| Basis domain for DNS entries. As a best practice, use an individual prefix together with the cluster domain of your base cluster.|`vedge.gcp.dev.k8s.jacksgrocerystore.com`|
```

Comment:
It is important to note here that access to the appropriate hosted zone must be available. Because with the base domain other dns entries will be created. May be a reference zo the dns section would be useful.

Reply:
@mandelsoft @Diaphteiros :
Diaphteiros says below that after the merge of [#850](https://github.com/gardener/gardener/pull/850) the key `hostedZoneID` might no longer be needed.

Will this comment still be relevant if we remove `hostedZoneID` ? - Thanks.